gdnative-core/src/core_types/vector2.rs

```diff
@@ -89,6 +90,17 @@ impl Vector2Godot for Vector2 {
         Vector2::new(self.y, -self.x)
     }
 
+    #[inline]
+    fn move_towards(self, to: Vector2, delta: f32) -> Self {
+        let vd = to - self;
+        let len = vd.length();
+        if len <= delta || approx::abs_diff_eq!(len, f32::EPSILON) {
```

Comment:
The first two arguments to the `abs_diff_eq` macro are `lhs` and `rhs`. Here you seem to want to compare `len` with` zero, so this should be `approx::abs_diff_eq!(0.0, len)`.

Reply:
changed